frame_idx: 0
margin: 0.2
perturb_delta: 0.01
test_num_x: 201
test_num_y: 201
gp:
    partition_on_hit_rays: false
    symmetric_partitions: true
    group_size: 26
    overlap_size: 6
    margin: 1
    init_variance: 1000000
    sensor_range_var: 0.01
    max_valid_range_var: 0.1
    occ_test_temperature: 30
    sensor_frame:
        valid_range_min: 0.2
        valid_range_max: 30
        angle_min: -2.356194496154785
        angle_max: 2.338737726211548
        num_rays: 270
        discontinuity_factor: 10
        rolling_diff_discount: 0.9
        min_partition_size: 5
    gp:
        kernel_type: erl::covariance::OrnsteinUhlenbeck<double, 1>
        kernel_setting_type: erl::covariance::Covariance<double>::Setting
        kernel:
            x_dim: 2
            scale: 1
            scale_mix: 1
            weights: [ ]
        max_num_samples: 256
    mapping:
        type: kInverseSqrt
        scale: 1
output_dir: gp_occ_regression
